### ISSUE TITLE
Screen reader text document doesn't show up in search

### ISSUE THREAD

**Issue by joedolson** [wpaccessibility/wp-a11y-docs#258]
I'm not sure why this is, but I'm unable to locate the CSS Screen Reader Text doc in search. It's possible there are other docs not in the search index, as well, if there's one, so it would be good to find out why and try and resolve it globally if possible.

https://wpaccessibility.org/docs/topics/code/screen-reader-text/

https://wpaccessibility.org/search/?q=screen-reader-text

- [x] I've checked the [existing issues for this change](https://github.com/wpaccessibility/wp-a11y-docs/issues)

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
